backend/btrixcloud/operator.py

```diff
@@ -364,7 +364,7 @@ async def update_crawl_state(self, redis, crawl, status):
             )
 
         if crawl.stopping:
-            await redis.set(f"{crawl.id}:stopping", "1")
+            await redis.set(f"{crawl.id}:stop-crawl", "1")
```

Comment:
```suggestion
            await redis.set(f"{crawl.id}:crawl-stop", "1")
```
matches the setting from webrecorder/browsertrix-crawler#303

Reply:
🤦🏻‍♀️ good catch, thanks!